how-to/workspace-platform-starter/CHANGELOG.md

```diff
@@ -3,6 +3,15 @@
 ## v14
 
 - Added customizable splash screen
+- Apps are refreshed based on the cache interval, if they change then the lifecycle event `apps-changed` is fired
+- Dock component subscribes to `apps-changed` lifecycle event and updates any dock based on their tagged apps
+- LaunchPage helper logic centralized for home, menus, dynamic dock and share
+- LaunchPage helper will always activate page if it already exists, unless `createCopyIfExists` is set
+- Added dynamic dock menu module which shows pages
```

Comment:
This module is an example of how you can use our popupWindow API for situations where you need a custom menu...something like that to make it clear this is an example.

Reply:
Updated